packages/babel-plugin/src/index.tsx

```diff
@@ -50,6 +50,9 @@ export default declare<State>((api) => {
             'leading',
             ` ${parseFilename(state.filename)} generated by ${pkgJson.name} v${version} `
           );
+
+          // Add a line break the comment
+          path.unshiftContainer('body', t.noop());
```

Comment:
oooo `babel` utilities `t.noop`. Nice.

Reply:
yeah you find wild things going through the api.. not documented anywhere of course